src/main/resources/org/openjdk/jextract/impl/resources/RuntimeHelper.java.template

```diff
@@ -90,24 +64,11 @@ final class RuntimeHelper {
     static MethodHandle upcallHandle(Class<?> fi, String name, FunctionDescriptor fdesc) {
         try {
             return MH_LOOKUP.findVirtual(fi, name, fdesc.toMethodType());
-        } catch (Throwable ex) {
+        } catch (ReflectiveOperationException ex) {
```

Comment:
I've left this `upcallHandle` method since we use it to initialize a class-level constant in the FI classes. It's useful to have the try/catch be out-of-line like this so we don't have to emit `static` blocks.

Reply:
I don't disagree. In fact I think I'm in favor of also leaving downcall handle (in both variants).